src/components/utils/CopyLinkButton.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @author sriram
+ *
+ * A common CopyLink menu item to use across Data and Apps
+ *
+ */
+import React from "react";
+import { useTranslation } from "i18n";
+import { build } from "@cyverse-de/ui-lib";
+import { IconButton, Tooltip } from "@material-ui/core";
+import { FileCopy } from "@material-ui/icons";
```

Comment:
I'm used to seeing a clipboard icon in places like GitHub for copy-to-clipboard actions, but the only MUI icon that looks like a clipboard is the `Assignment` icon 🤔 

This `FileCopy` icon looks good in this PR's screenshots, though 👍